cmd/cli/cmd/manifest/generate/attributes/attribute.go

```diff
@@ -3,9 +3,10 @@ package attributes
 import (
```

Comment:
Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 26